app/core/Tracker2/LegacyTracker.js

```diff
@@ -20,12 +20,12 @@ export default class LegacyTracker extends BaseTracker {
   async _initializeTracker () {
     this.legacyTracker = new CocoLegacyTracker()
 
-    this.cookieConsent.on('change', () => {
-      this.legacyTracker.cookies = this.cookieConsent.getStatus()
-    })
-
+    this.legacyTracker.cookies = this.store.state.tracker.cookieConsent
+    this.store.watch(
+      (state) => state.tracker.cookieConsent,
```

Comment:
Can we use the getter here similar to BaseTracker?

Reply:
Great question, I opted to use the raw state here to keep the functionality in the legacy tracker 100% the same.  The legacy tracker tracks all three state variables in it's `this.cookies` member variable.  From what I can tell it only uses the declined parameter but I think it's safer to keep 100% the same especially because the use of this legacy tracker and the binding between them will go away in the coming weeks (it's just a temporary measure to allow us to complete part of the refactor and then launch and test).

Reply:
That makes sense.